src/data.js

```diff
@@ -10,7 +10,7 @@
     not by your logic
     not by your listeners
 */
-export const prizes = ['100','200','500']
+export const prizes = ['100','200','500', '1000', '2000', '2000', '2000', '2000', '2000', '10000'];
```

Comment:
Thuis change seems not really correct or relevant for the current feature 🤷‍♂️

Reply:
We updated the prizes. It can be also seen on the breakpointPage.